app/Repositories/User/UserRepository.php

```diff
@@ -2,12 +2,11 @@
 
 namespace App\Repositories\User;
 
+use App\Http\Resources\V1\NotificationListResource;
 use App\Repositories\BaseRepository;
-use App\Repositories\User\UserRepositoryInterface;
```

Comment:
Why did you take this line out? Are you implementing UserRepositoryInterface anymore?

Reply:
As both are in same directory so it is not needed to use interface here. BTW this line was removed by editor code optimization feature.

Reply:
Acknowledged!